release.sh

```diff
@@ -53,10 +52,12 @@ git fetch -q "${REPOSITORY}"
 RELEASE_DATE=$(date -u '+%Y-%m-%d')
 git checkout -b "prepare-release-${RELEASE_DATE}" "${REPOSITORY}/${BRANCH_FROM}"
 
-#Disable the shell check as the colour codes only work with interpolation.
-# shellcheck disable=SC2059
-printf "Validating the build is ${GREEN}green${NC}"
-mvn -q clean verify
+if [[ -z "${SKIP_VALIDATION:-}" ]]; then
```

Comment:
`SKIP_VALIDATION=false ./release.sh` would lead to the the validation being skipped, which is weird.  I'd declare and default the value up front to be false, then make this if look for true.

Reply:
Good point! Will fix.

Reply:
fixed in f139f179c6f39b6f2c3ee3726b63d9f80a2230f8